types/jquery/JQuery.d.ts

```diff
@@ -9837,7 +9840,14 @@ $( "button" ).on( "click", function() {
 </html>
 ```
      */
-    replaceWith(newContent_function: JQuery.htmlString | JQuery | JQuery.TypeOrArray<Element> | ((this: TElement) => any)): this;
+    replaceWith(newContent_function: JQuery.htmlString |
+                                     JQuery<JQuery.Node> |
```

Comment:
This seems strange to me.

Reply:
The change here is from `JQuery` (`JQuery<HTMLElement>`) to `JQuery<JQuery.Node>`. `.replaceWith()` is indeed more permissive than previously declared and can accept `Text` and `Comment` objects. See [these tests](https://jsfiddle.net/leonard_thieu/ysa502gL/16/) for verification.